festipals/src/components/PalsPage.js

```diff
@@ -10,7 +11,8 @@ export default class PalsPage extends Component {
         <div className="container">
           <h1>Your Pals</h1>
           <p>Click on a Pal&rsquo;s name to see his or her personal schedule</p>
-          <Pals />
+          {/*} looping through all the acts in the sample data file array to display the acts */}
+            {SampleDataPals.acts.map(function (pal) { return <Pals key={pal._id} id={pal._id} allPals={SampleDataPals} first_name={pal.first_name}  /> }) }
```

Comment:
1. Here you are mapping the acts instead of the pals array.
2. Also if we want to stay consistent with the naming, i think we should use camelCase for props, so firstName and snake_case in the database (as it is). But that's not something that's wrong, it's just my opinion because as you may noticed I'm can be a little perfectionistic at times. ;)
3. I think we also need to pass it the last name, as there can be many people named the same and we decided not to use profile pictures if i recall that correctly. (it would be a pain in the ass to store and format those)